### ISSUE TITLE
Randomize .mov file when new joke is generated

### ISSUE THREAD

**Issue by randydaniel** [randydaniel/dadjoke.us#8]
AAUIW to see a new .mov file when a new joke is generated.

- .mov files in assets/video folder

**Comment by kanghamin** [comment]
I tried to tackle this issue with changing state, but I failed setting up the state inside of functional component.. 

My approach was:
0. Import all the videos
1. Setting state with video - this.state({ video: null})
2. Creating random video function with array and random number - [video1, video2, video3][randomNum]
3. Manipulating the state with random video function.
4. Changing the state when onclick is fired.

**Comment by randydaniel** [comment]
> I tried to tackle this issue with changing state, but I failed setting up the state inside of functional component..
> 
> My approach was:
> 0. Import all the videos
> 
>     1. Setting state with video - this.state({ video: null})
> 
>     2. Creating random video function with array and random number - [video1, video2, video3][randomNum]
> 
>     3. Manipulating the state with random video function.
> 
>     4. Changing the state when onclick is fired.

Thanks for taking a look at this @kanghamin -- I'll give it another shot, but it sounds exactly like what my initial attempt was.

**Comment by kanghamin** [comment]
@randydaniel Okay, I did little research and found that class component only allows having state. Functional component needs Hooks in order to have state, so I fixed this issue by adding Hook. It seems like I have authentificational issue, so I couldn't create repo for it. I will update two screenshots instead. 

<img width="1187" alt="Screen Shot 2020-01-25 at 1 06 57 PM" src="https://user-images.githubusercontent.com/38988290/73126751-a62d5880-3f73-11ea-82b7-1f582b705a4a.png">
<img width="1182" alt="Screen Shot 2020-01-25 at 1 07 07 PM" src="https://user-images.githubusercontent.com/38988290/73126749-a62d5880-3f73-11ea-86f2-f10418097821.png">
